.github/workflows/release.yaml

```diff
@@ -107,6 +111,8 @@ jobs:
     - name: Build & Push Artifacts
       env:
         REF: ${{ github.event.ref }}
+        BUILD_NUMBER: ${{ github.run_attempt }}
```

Comment:
I think you need a combination of run_number and run_attempt because I think run_attempt will always be 1 for the first attempt.

Reply:
That's fine since the ref/tag should be different right?
Attempts only rev if we rebuild the same tag.